CodingTracker/CodingTracker/Program.cs

```diff
@@ -0,0 +1,12 @@
+﻿using CodingTrackerLibrary;
+using CodingTracker;
+
+SetupDatabase setupDatabase = new SetupDatabase(Settings.GetConnectionString());
+setupDatabase.InitializeDatabase();
+
+#if DEBUG
```

Comment:
I would recommend not to use compiler logic if possible. Could this not have been stored in the config with the connection string?